media/js/firefox/whatsnew/whatsnew-42.js

```diff
@@ -14,7 +14,7 @@
         });
     }
 
-    Mozilla.HighlightTarget.init('.button.dark');
+    Mozilla.HighlightTarget.init('.button.button-dark');
 
     $('.button.dark').on('highlight-target', function() {
```

Comment:
I think this makes sense to try and separate. There is also the option for never using ID's in our CSS and reserving them only for JS hooks, which is a rule a lot of other sites tend to follow these days. Our base CSS is a little too dependent on ID's to make this an easy change though.